olsrd/Makefile

```diff
@@ -34,7 +34,7 @@ endef
 define Package/olsrd
   $(call Package/olsrd/template)
   MENU:=1
-  DEPENDS:=+libpthread +libubus
+  DEPENDS:=+libpthread +libubus +iptables-nft +ip6tables-nft
```

Comment:
This breaks compatibility with fw3.  By quick looking at commit https://github.com/openwrt/openwrt/commit/795e7155cbe3e78669f6821bf7aecb7c4e1e1afb it should be enough to add iptables and iptables6 w/o any suffix. :)